app/electron/createProject.js

```diff
@@ -0,0 +1,46 @@
+const sh = require("shelljs");
+const log = require("electron-log");
+
+// Examples
+/*
+let project1 = {
+  name : "hello1",
+  type : "node"
+}
+*/
+
+module.exports = (project, location) => {
+  // Making a Projects Folder to store all the projects
+  let cmd = `${location[0]}: && cd ${location} && mkdir Projects`;
+
+  sh.exec(cmd, { silent: true }, (code, stdout, stderr) => {
+    if (code !== 0) {
+      log.error(stderr);
+    } else if (code === 0) {
+      log.info("Projects Folder successfully created");
+    }
+  });
+
+  if (project.type === "node") {
+    // Traversing to inside the Projects Folder
+    cmd = `mkdir ${project.name} && cd ${project.name} && npm init -y`;
+  } else if (project.type === "django") {
+    cmd = `django-admin startproject ${project.name} && cd ${project.name} && echo > requirements.txt`;
+  }
```

Comment:
Take this as param rather than determining in the function

Reply:
Could you please elaborate on what and how exactly are we planning to take data from the frontend?